src/commands/ticket.ts

```diff
@@ -0,0 +1,68 @@
+import { ApplyOptions } from '@sapphire/decorators';
+import { Command, CommandOptionsRunTypeEnum } from '@sapphire/framework';
+import {
+    ActionRowBuilder,
+    ModalBuilder,
+    TextInputBuilder,
+    TextInputStyle,
+    EmbedBuilder,
+    type ChatInputCommandInteraction,
+} from 'discord.js';
+import { Color } from '../lib/embeds';
+import {InputId, ModalId} from '../lib/ticket';
+
+const error = (interaction: ChatInputCommandInteraction, content: string) => {
+    return interaction.followUp({
+        embeds: [new EmbedBuilder().setColor(Color.Red).setDescription(content)],
+        ephemeral: true,
+    });
+};
+
+@ApplyOptions<Command.Options>({
+    description: 'Allow users to create a ticket from anywhere',
+    runIn: [CommandOptionsRunTypeEnum.GuildAny],
+})
+export class SelfTimeoutCommand extends Command {
+    public override async chatInputRun(interaction: ChatInputCommandInteraction) {
+        if (!interaction.inGuild()) {
+            await error(interaction, 'Command only available in servers');
+            return;
+        }
+
+        const ticketModal = new ModalBuilder()
+            .setCustomId(ModalId.Ticket)
+            .setTitle('Enter your information for verification')
```

Comment:
Leftover from copy/paste. Not sure what would be best here.
```suggestion
            .setTitle('Ticket')
```